electrum/gui/qt/wizard/wallet.py

```diff
@@ -1357,7 +1357,7 @@ def on_ready(self):
 
         device_id = _info.device.id_
         client = self.plugins.device_manager.client_by_id(device_id, scan_now=False)
-        if not client.handler:
+        if not getattr(client, 'handler', None):
             client.handler = self.plugin.create_handler(self.wizard)
```

Comment:
This is clearly incorrect... If client is None, then `getattr(client, 'handler', None)` will be None, and `if not None` will become `if True`, so the branch will execute, and then already on the next line `client.handler` will error out...